packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md

```diff
@@ -238,7 +238,7 @@ class BasicNotificationDrawer extends React.Component {
       </DropdownGroup>
     ];
     const headerToolbar = (
-      <Toolbar>
+      <Toolbar isFullHeight>
```

Comment:
I believe the Dropdowns themselves also need the isFullHeight modifier on them to make the height of the user dropdowns consistent between examples

Reply:
I think that's like line 874

Reply:
It does look to make any visual change and 7/12 examples for username dont have that (searched for `ned username` and `john smith`) so should we change all example to have it or remove it from existing examples?

Reply:
@mcarrano 
Should the user dropdown in mastheads in our demos always be full height? Looks like in the react workspace right now, only half of them are full height.

Reply:
Yes, I believe so, i.e. when they are open, the selected highlight is at the bottom of the masthead.

Reply:
Oh I see, I also counted the examples for `pageHeader`. 
Added `isFullHeight`